Client/Frontend/Strings.swift

```diff
@@ -222,3 +222,58 @@ extension Strings {
+    
+    public static let safeBrowsingPhishingRisks1 = NSLocalizedString("SafeBrowsingPhishingRisks_1", bundle: Bundle.shared, value: "or, if you understand the risks to your security,", comment: "Risk")
+    public static let safeBrowsingPhishingVisitUnsafe = NSLocalizedString("SafeBrowsingPhishingVisitUnsafe", bundle: Bundle.shared, value: "visit this unsafe site", comment: "Risk")
+    
+    //Unwanted Software
+    public static let safeBrowsingUnwantedSoftwarePageHeader = NSLocalizedString("SafeBrowsingUnwantedSoftwarePageHeader", bundle: Bundle.shared, value: "Harmful site ahead", comment: "Page header")
+    public static let safeBrowsingUnwantedSoftwareErrorDescription = NSLocalizedString("SafeBrowsingUnwantedSoftwareErrorDescription", bundle: Bundle.shared, value: "Attackers on %@ might attempt to trick you into installing programs that harm your browsing experience (for example, by changing your homepage or showing extra ads on sites you visit).", comment: "Error Description")
+    public static let safeBrowsingUnwantedSoftwareErrorDetection = NSLocalizedString("safeBrowsingUnwantedSoftwareErrorDetection", bundle: Bundle.shared, value: "Google Safe Browsing recently", comment: "Error Detection")
+    public static let safeBrowsingUnwantedSoftwareErrorDetectionLinkText = NSLocalizedString("SafeBrowsingUnwantedSoftwareErrorDetectionText", bundle: Bundle.shared, value: "found harmful programs", comment: "Error Detection Link Text")
+    public static let safeBrowsingUnwantedSoftwareErrorDetectionPage = NSLocalizedString("SafeBrowsingUnwantedSoftwareErrorDetectionPage", bundle: Bundle.shared, value: "on %@.", comment: "Error Detection Link")
+    public static let safeBrowsingUnwantedSoftwareRisks1 = NSLocalizedString("SafeBrowsingUnwantedSoftwareRisk_1", bundle: Bundle.shared, value: "If you understand the risks to your security, you may", comment: "Risk")
+    public static let safeBrowsingUnwantedSoftwareVisitUnsafe = NSLocalizedString("SafeBrowsingUnwantedSoftwareVisitUnsafe", bundle: Bundle.shared, value: "visit this site", comment: "Risk")
+    public static let safeBrowsingUnwantedSoftwareRisks2 = NSLocalizedString("SafeBrowsingUnwantedSoftwareRisk_2", bundle: Bundle.shared, value: "before the harmful programs have been removed.", comment: "Risk")
+    
+    //Potentially Harmful Application
+    public static let safeBrowsingHarmfulApplicationPageHeader = NSLocalizedString("SafeBrowsingHarmfulApplicationPageHeader", bundle: Bundle.shared, value: "Harmful site ahead", comment: "Page header")
+    public static let safeBrowsingHarmfulApplicationErrorDescription = NSLocalizedString("SafeBrowsingHarmfulApplicationErrorDescription", bundle: Bundle.shared, value: "Google Safe Browing blocked this page because it might try to install dangerous apps that steal or delete your information (for example, photos, passwords, messages and credit cards).", comment: "Error Description")
```

Comment:
Sorry if I've commented on this already, but I can't find my notes and I also can't find a test page for this type of blocking. 

I'd like to suggest that we match the style we see in the other messages, so how about:

> Attackers on %@ might try to install dangerous apps that steal or delete your information (for example, photos, passwords, messages and credit cards).